core/src/progress_map.rs

```diff
@@ -14,7 +14,9 @@ use std::{
     sync::{Arc, RwLock},
 };
 
-pub(crate) type LockoutIntervals = BTreeMap<Slot, Vec<(Slot, Rc<Pubkey>)>>;
+type VotedSlot = Slot;
```

Comment:
Even if I say this, rustc still use `u64` due to https://github.com/rust-lang/rust/issues/17164. But better than nothing. :)